src/test/fuzz/key.cpp

```diff
@@ -305,4 +306,25 @@ FUZZ_TARGET_INIT(key, initialize_key)
             assert(key == loaded_key);
         }
     }
+
+    {
+        std::array<uint8_t, 32> rnd32;
+        memcpy(rnd32.data(), &random_uint256, 32);
```

Comment:
random_uint256 is uniformly distributed, making it harder for the fuzz engine to discover potential edge cases

Reply:
Makes sense. Fixed. I ended up combining the encode/decode fuzzing.